pkg/tfgen/generate.go

```diff
@@ -95,6 +96,19 @@ func (l language) emitSDK(pkg *pschema.Package, info tfbridge.ProviderInfo, outD
 				return nil, err
 			}
 		}
+
+		// We exclude the "tests" directory because some nodejs package dirs (e.g. pulumi-docker)
+		// store tests here. We don't want to include them in the overlays because we don't want it
+		// exported with the module, but we don't want them deleted in a cleanup of the directory.
+		exclusions := codegen.StringSet{}
+		exclusions.Add("tests")
```

Comment:
You can use the `NewStringSet` factory function here:

```suggestion
		exclusions := codegen.NewStringSet("tests")
```